minos/common/model/declarative/entities.py

```diff
@@ -0,0 +1,28 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+
+from typing import (
+    Optional,
+)
+from uuid import (
+    UUID,
+    uuid4,
+)
+
+from .abc import (
+    DeclarativeModel,
+)
+
+
+class Entity(DeclarativeModel):
+    uuid: UUID
```

Comment:
Could you include at least a small line of documentation for the `Entity` class.